libimage/runtime_test.go

```diff
@@ -56,6 +58,13 @@ func testNewRuntime(t *testing.T, options ...testNewRuntimeOptions) *Runtime {
 	return runtime
 }
 
+func testRuntimePullImage(t *testing.T, r *Runtime, ctx context.Context, imageName string) {
+	pullOptions := &PullOptions{}
+	pullOptions.Writer = os.Stdout
```

Comment:
(Absolutely non-blocking: This can be directly embedded in the literal,
```go
&PullOptions{
    Writer: …
}
```
and I’d personally just embed it into the `Pull` call site instead using a variable, but that’s totally just a personal preference.)

Reply:
I've tried it, but it has a problem: `unknown field Writer in struct literal of type PullOptions`

Reply:
Ah, it would have to be
```go
&PullOptions{
   CopyOptions { Writer: … }
}
```
… and then it’s not an improvement.